Xamarin.PropertyEditing.Mac/Controls/Layout.cs

```diff
@@ -3,6 +3,6 @@ namespace Xamarin.PropertyEditing.Mac
 {
 	internal static class Layout
 	{
-		internal const float GoldenRatioLeft = .38196601125f;
```

Comment:
Why was this changed? This will affect a number of things and I'm not sure how you arrived at the new number, but I'm pretty sure it's incorrect.

Reply:
I was playing around with the left margin and forget to reset it back. Will be fixed in next push.